rpc/src/rpc_subscriptions.rs

```diff
@@ -1154,7 +1154,6 @@ impl RpcSubscriptions {
                 num_signatures_found.load(Ordering::Relaxed),
                 num_signatures_notified.load(Ordering::Relaxed),
             );
-            inc_new_counter_info!("rpc-subscription-notify-bank-or-gossip", total_notified);
```

Comment:
This one actually isn't being sent to metrics in the datapoint. Did you decide to skip because we can just sum the 4 notification types? That seems perfectly reasonable to me, just clarifying.

Reply:
Correct, it can be derived from other things and I thought the name was slightly "off" for what the actual value was, so I decided to drop it